app/policies/user_reduction_policy.rb

```diff
@@ -21,6 +21,11 @@ def update?
     credential.project_ids.include?(record.reducible.project_id)
   end
 
+  def current_user_reductions?
```

Comment:
Scope up above should probably include `.or(self.scope.where(user_id: credential.user_id)`

Reply:
Thank you, this was a big help.